### PR TITLE
release: 1.14.4

### PR DESCRIPTION
**Pull request by jasonbahl**
# Release Notes

### New Features

- [#2826](https://github.com/wp-graphql/wp-graphql/pull/2826): feat: pass connection config to connection field

### Chores / Bugfixes

- [#2818](https://github.com/wp-graphql/wp-graphql/pull/2818): chore: update webonyx/graphql-php to v14.11.9. Thanks @justlevine!
- [#2813](https://github.com/wp-graphql/wp-graphql/pull/2813): fix: replace double negation with true. Thanks @cesarkohl!

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -1,10 +1,10 @@
 === WPGraphQL ===
 Contributors: jasonbahl, tylerbarnes1, ryankanner, hughdevore, chopinbach, kidunot89
-Tags: GraphQL, API, Gatsby, Headless, Decoupled, React, Nextjs, Vue, Apollo, REST, JSON,  HTTP, Remote, Query Language
+Tags: GraphQL, JSON, API, Gatsby, Faust, Headless, Decoupled, Svelte, React, Nextjs, Vue, Apollo, REST, JSON, HTTP, Remote, Query Language
 Requires at least: 5.0
-Tested up to: 6.1
+Tested up to: 6.2
```
Comment (justlevine) [inline readme.txt:5]:
```suggestion
Tested up to: 6.2.2
```

File: composer.json [hunk 2]
```diff
@@ -44,8 +44,9 @@
 		"phpunit/phpunit": "^9.5",
 		"simpod/php-coveralls-mirror": "^3.0",
 		"slevomat/coding-standard": "^8.9",
-		"szepeviktor/phpstan-wordpress": "~1.2.0",
-		"wp-graphql/wp-graphql-testcase": "~2.3"
+		"szepeviktor/phpstan-wordpress": "~1.3.0",
+		"wp-graphql/wp-graphql-testcase": "~2.3",
+		"wp-cli/wp-cli": "2.7.1"
```
Comment (justlevine) [inline composer.json:49]: @jasonbahl v2.8.1 is out so its safe to remove `wp-cli/wp-cli` again

Comment (jasonbahl) [reply composer.json:49]: Thanks! will update

File: .github/workflows/schema-linter.yml [hunk 3]
```diff
@@ -14,7 +14,7 @@ jobs:
     name: Lint WPGraphQL Schema
     services:
       mariadb:
-        image: mariadb
+        image: mariadb:latest
```
Comment (justlevine) [inline .github/workflows/schema-linter.yml]: You want it `mariadb:10`. 11.x is incompatible with WP 6.2.2 (in all places)

**Pull request merged**